src/ShopProductExportBundle/Interfaces/ShopProductExportHandlerInterface.php

```diff
@@ -13,11 +13,30 @@
 
 interface ShopProductExportHandlerInterface
 {
+    /**
+     * do any initialization work that needs to be done before you want to run the export.
+     *
+     * @return void
+     */
     public function Init();
 
+    /**
+     * @param \TdbShopArticleList $oArticleList
+     * @return void
+     */
     public function SetArticleList(\TIterator $articleList);
 
+    /**
+     * @param bool $debug
+     * @return void
+     */
     public function SetDebug($debug);
 
+    /**
+     * Run the export. returns true if the export was successful, otherwise false
+     * this method should not be overwritten in child classes.
```

Comment:
If that is true we could add a `fixme` to make this final in the future

Reply:
"This method should not be overwritten in child classes" makes no sense at all on an interface. I must have copy & pasted the whole docblock from it's implementation `TPkgShopProductExportBaseEndPoint`, where this comment could make sense.

I

* removed the comment from the interface
* Added `@final` to the docblock of `TPkgShopProductExportBaseEndPoint`. This does not change runtime behaviour but lets tools know that they shouldn't extend the method.